### ISSUE TITLE
cleaned by antivirus

### ISSUE THREAD

**Issue by robertwt7** [finndev/PokeBuddy#667]
it's detected as suspicious by eset antivirus :s, is there anyway to solve this?

**Comment by XenSide** [comment]
Yes, one easy fix is not using trash antiviruses (i.e Avast, AVG or ESET).
If you want a real antivirus use your brain, that's enough, and if you're dumb and not able to do that then try something like Kaspersky.

**Comment by curtiola** [comment]
Hahaha holy hostility, but yeah he's right, you don't need to be a genius to avoid downloading viruses

**Comment by harambeman52** [comment]
Use a good antivirus
Like malwarebytes or ccleaner

**Comment by XenSide** [comment]
Ccleaner is NOT an antivirus and you should not use it as one, Malwarebytes is not bad tho.
And yes, I'm very hostile when i read things like this because i think using things like avast or ESET is really unacceptable in 2016, either you use a good one or you use the best one: Your brain.
